modules/sor/sorV2/lib/pools/composableStable/composableStablePool.ts

```diff
@@ -327,10 +304,10 @@ export class ComposableStablePool implements BasePool {
 
         if (swapKind === SwapKind.GivenIn) {
             // Return max valid amount of tokenIn
-            return (tIn.amount * WAD) / tIn.rate;
+            return (tIn.amount * tIn.rate) / WAD;
         }
         // Return max amount of tokenOut - approx is almost all balance
-        return (tOut.amount * WAD) / tOut.rate;
+        return (tOut.amount * tOut.rate) / WAD;
```

Comment:
Was this incorrect in the previous code? Did you identify it from tests?

Reply:
Yes, I identified this comparing with the old SOR

Reply:
I'm looking at the old SOR and it seems to divide by the token rate, instead of multiplying by it [here](https://github.com/balancer/balancer-sor/blob/de36edc1bd110a3db1160e55ba5e8abace43a621/src/pools/metaStablePool/metaStablePool.ts#L171)
Am I missing something?